frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.tsx

```diff
@@ -61,9 +58,10 @@ export const PublicOrEmbeddedQuestion = ({
 
   useMount(async () => {
     if (uuid) {
-      setPublicQuestionEndpoints(uuid);
+      overrideRequestsForPublicOrStaticEmbeds("public");
```

Comment:
We use `usePublicDashboardEndpoints` for dashboards, we should do the same here for consistency. I was talkinag about the way we extract these code in `useMount` into a hook btw.